### PR TITLE
Fix references from develop to 1.7.8.0

### PR DESCRIPTION
**Pull request by thomasnares**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.x
| Description?  | Replacement of all references to the PS repo from develop to 1.7.8.0 for 1.7.x doc

### CODE REVIEW

**Review by kpodemski — DISMISSED** [review]
Thanks, @thomasnares 

A few things:
- ~It should be `1.7.8.x`~ it's ok to link to the tag, but I'm not sure if to `1.7.8.0` or recent version :-)
- please double-check if links with `#` point to the place in the code relevant to what is written

File: themes/reference/templates/notifications.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ In the "Classic" Theme, [notifications are implemented as a partial template fil
 
 ## Add your own message in your front controller
 
-Your front controller holds [the 4 following variables](https://github.com/PrestaShop/PrestaShop/blob/develop/classes/controller/FrontController.php#L616-L621):
+Your front controller holds [the 4 following variables](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.0/classes/controller/FrontController.php#L616-L621):
```
Comment (kpodemski) [inline themes/reference/templates/notifications.md]: Wrong link

File: themes/reference/templates/notifications.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ Your front controller holds [the 4 following variables](https://github.com/Prest
 
 They are PHP arrays, and they hold messages as a string.
 
-Since PrestaShop 1.7, you can [redirect the customer AND display a message after an action](https://github.com/PrestaShop/PrestaShop/blob/develop/classes/controller/FrontController.php#L614-L633).
+Since PrestaShop 1.7, you can [redirect the customer AND display a message after an action](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.0/classes/controller/FrontController.php#L614-L633).
```
Comment (kpodemski) [inline themes/reference/templates/notifications.md]: Wrong place in the code highlighted

File: development/architecture/migration-guide/hooks.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ Getting the list of available Hooks in modern pages is really easy. Thanks to th
 
 Use this trick to find out which hooks are called on a legacy page.
 
-In ``classes/Hook``, find the [exec() function](https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Hook.php#L733) and add the following code:
+In ``classes/Hook``, find the [exec() function](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.0/classes/Hook.php#L733) and add the following code:
```
Comment (kpodemski) [inline development/architecture/migration-guide/hooks.md]:
```suggestion
In ``classes/Hook``, find the [exec() function](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.0/classes/Hook.php#L766) and add the following code:
```

**Review by kpodemski — CHANGES_REQUESTED** [review]
Some changes are needed.

@eternoendless, shall we link to the most recently released version? e.g.: `1.7.8.7`?

**Comment by eternoendless** [comment]
> shall we link to the most recently released version? e.g.: `1.7.8.7`?

I think the latest .0 should do, unless the code is not yet present in that version. Otherwise we'd have to update all links whenever we publish a new patch version.

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thanks @thomasnares :))

**Pull request merged**